index.js

```diff
@@ -51,6 +51,7 @@ const createClient = (profile, userAgent, opt = {}) => {
 			// todo: for arrivals(), this is actually a station it *has already* stopped by
 			direction: null, // only show departures stopping by this station
 			duration: 10, // show departures for the next n minutes
+			results: 50, // max. number of results – `null` means "whatever HAFAS wants"
```

Comment:
There seems to be an endpoint-specific maximum value *anyways*. My intention behind `50` was to make the default – people who don't care about options & performance, who just want to get some departures – not too wasteful. People who care can just pass `1000` or `Infinity` if they want more departures.

Reply:
I'd leave this to the endpoint to decide. Otherwise you would, in addition to the endpoint's default, also get the profile implementor's choice as another layer of abstract "other people's choice".

Reply:
Let‘s keep the option but set the default to `null`.

Reply:
This change shouldn't be in this PR, as it is on `master` already. I will adapt this PR to `null` anyways.